src/module/actor/mixins/actor-conditions.js

```diff
@@ -1,6 +1,12 @@
 import { SFRPG } from "../../config.js"
 
 export const ActorConditionsMixin = (superclass) => class extends superclass {
+    /**
+     * Check if the Actor has the condition.
+     * @param {String} conditionName The name of the condition. Must match any key from config.js SFRPG.statusEffectIconMapping. Case insensitive.
```

Comment:
The conditionName is suggested to be case insensitive in your comments, but it is case sensitive.

Reply:
Yes, you are correct. Even though there are comparisons made in lower case only there are also checks that are sensitive. I'll change the documentation.